### DISCUSSION TITLE
Multiple submissions?

### DISCUSSION THREAD

**Discussion by abea** [cssbattle/forum#241] [Q&A] [answered]
Before I submit, I'm wondering if we can make multiple submissions. It'd be nice to see my score then keep updating to improve it, but I don't want to get stuck with my first score if it's not totally optimized yet.

**Comment by gunnarbittersmann** [comment, marked as answer]
Yes, you can.
Fun fact: right of now, I’ve done 2479 submissions on [target #12](https://cssbattle.dev/play/12).

**Reply by abea** [reply]
Thanks!

**Reply by gunnarbittersmann** [reply]
2624 submissions now. I couldn’t find a shorter solution, meh.
